src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -70,6 +70,7 @@ public function iAmLoggedInAs($email)
 
     /**
      * @Given /^I am logged in customer$/
+     * @Given /^I am logged in as a customer$/
```

Comment:
Great! 👍  So still this change should not be done here, if it's done in mentioned PR 😄